task/gcloud/0.1/gcloud.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   params:
   - name: gcloud-image
     description: gcloud CLI container image to run this task
-    default: google/cloud-sdk:slim
+    default: gcr.io/google.com/cloudsdktool/cloud-sdk:slim@sha256:70d3c9ef711c704259b06485f9ab3bab6e1b5c99e4a5c1ed37f9338004664c17 #tag: slim, 312.0.0-slim
```

Comment:
@vdemeester  @chmouel  Are we fine with image registery change?

Reply:
all good yes,

Reply:
yes :wink: